source/configuration_manual/config_file/config_variables.rst

```diff
@@ -54,6 +48,32 @@ If :ref:`var_expand_crypt_plugin` is loaded, these also work globally:
 |                               | .. dovecotadded:: 2.2.29    |
 +-------------------------------+-----------------------------+
 
+.. _variables-system-variables:
+
+Supported system variables
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``cpu_count``
+  Number of CPUs available. Works only on Linux and FreeBSD-like systems.
+``hostname``
+  Hostname (without domain). Can be overridden with DOVECOT_HOSTNAME
+  environment variable. NOTE: This is overridden for
+  :ref:`mail user variables <variables-mail_user>`.
```

Comment:
I don't think system:hostname is overridden? It could maybe point to the %{hostname} in there, and vice versa.